js/parts/Html.js

```diff
@@ -24,11 +26,19 @@ var attr = H.attr,
     win = H.win,
     wrap = H.wrap;
 
-// Extend SvgElement for useHTML option
+// Extend SvgElement for useHTML option.
 extend(SVGElement.prototype, /** @lends SVGElement.prototype */ {
+
     /**
      * Apply CSS to HTML elements. This is used in text within SVG rendering and
      * by the VML renderer
+     *
+     * @private
+     * @function Highcharts.SVGElement#htmlCss
```

Comment:
Is it necessary to explicitly write the function name when we have the `@lends` statement plus the function name in the code?

Reply:
I find it necessary to improving the understanding of the code structure and also solve some JSDoc errors. For consistency I do it everywhere, even when JSDoc is getting it right.

You can of course just parse code and try to extract the meaning behind it, but JSDoc is failing especially determing static and instance functions, and sometimes even placing it at the correct class/namespace.

So in explicitly describing the function relation there is no room left for JSDoc to guess.